src/components/Header/index.js

```diff
@@ -42,6 +46,10 @@ class Header extends React.Component {
             <li>
               <a href="#!">Favorites</a>
             </li>
+            <li className="divider" tabIndex="-1" />
+            <li>
+              <a href="#!" onClick={this.logout}>Logout</a>
```

Comment:
@Tittoh Shouldn't the page be directed to the homepage after a successful logout

Reply:
Fixed it